Helper/StoreConfig.php

```diff
@@ -355,6 +368,11 @@ function ($store) {
                 return $store->isActive();
             }
         );
+    } 
+
+    public function getWebSiteName()
+    {
+        return $this->getCurrentStore()->getName();
```

Comment:
The name of the function does not correspond to it's implementation. Is it being used?

Reply:
same here